tox.ini

```diff
@@ -67,5 +67,5 @@ directory = cover
 [flake8]
 max-complexity = 10
 exclude = djstripe/migrations/, .tox/, build/lib/, .venv/
-ignore = W191, W503, E203, E501
+ignore = W191, W503, E203
```

Comment:
Remove this commit; the ignore is there because black does this for us, it's very much intended. flake8 is too aggressive whereas black is more lenient.